api/data_refinery_api/views.py

```diff
@@ -107,18 +107,20 @@ def list(self, request, *args, **kwargs):
         response.data['filters']['publication'] = {}
         response.data['filters']['organism'] = {}
 
-        techs = self.get_queryset().values('technology').annotate(Count('technology', unique=True))
+        qs = self.filter_queryset(self.get_queryset())
```

Comment:
Is there a way to access this without calling the filter? Or does it not matter since the QS isn't calculated until serialization time.